packages/contracts/contracts/voting/majority/MajorityVotingBase.sol

```diff
@@ -347,8 +361,16 @@ abstract contract MajorityVotingBase is
         supportThreshold = _supportThreshold;
         minParticipation = _minParticipation;
         minDuration = _minDuration;
+        minProposerVotingPower = _minProposerVotingPower;
+
+        emit VoteSettingsUpdated({
+            minParticipation: _minParticipation,
+            supportThreshold: _supportThreshold,
+            minDuration: _minDuration,
+            minProposerVotingPower: _minProposerVotingPower
+        });
     }
 
     /// @notice This empty reserved space is put in place to allow future versions to add new variables without shifting down storage in the inheritance chain (see [OpenZepplins guide about storage gaps](https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps)).
-    uint256[47] private __gap;
+    uint256[46] private __gap;
```

Comment:
Do we really need to decrement this value? We haven't started shipping and upgrading plugins

Reply:
We should always maintain the storage gaps when we add / remove storage variables

Reply:
Even before we release anything? When we deploy the core again, existing test DAO's will need to be created again

Reply:
What would be the benefit of not adapting the storage gap?

Ping @novaknole

Reply:
We should maintain this now ! don't see any other way. `test DAO` Jordi mentions will already have a problem as currently we don't pay attention to variable ordering.